<launch>

  <arg name="app_filepath"/>
  <arg name="ros_frame_id"/>

  <node name="program_verifier" pkg="evd_ros_core" type="program_verifier.py" output="screen">

  </node>

  <node name="grasp_verifier" pkg="evd_ros_core" type="grasp_verifier.py" output="screen">
    <param name="app_filepath" value="$(arg app_filepath)"/>
  </node>

</launch>
